atd-vzv/src/constants/populationEstimates.js

```diff
@@ -0,0 +1,26 @@
+export const populationEstimates = [
```

Comment:
It might make the references more readable to use an object here for `populationEstimates` instead of an array of objects. That way you could reference estimates with something `populationEstimates["years"]["2020"]` instead of relying on arbitrary indexes. Just a suggestion since this works as it is.

Maybe something like:
```
{
    "years": {
       "2020": 989233,
    },
    "sourceURL": <the url>,
    "sourceString": <the string>,
}
```